Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Search.hs

```diff
@@ -153,6 +156,17 @@ handler merchant sReq = do
 
   allFarePoliciesProduct <- getAllFarePoliciesProduct merchantId merchantOpCityId fromLocationLatLong toLocationLatLong
   let farePolicies = selectFarePolicy result.distance allFarePoliciesProduct.farePolicies
+
+  customerCancellationDue <- case sReq.customerPhoneNum of
```

Comment:
Add a merchantOperatingCity level check to enable or disable the cancellation dues.